docs/reference/manifest/webapplicationinfo.md

```diff
@@ -5,6 +5,9 @@ Supports single sign-on (SSO) in Office Add-ins. This element contains informati
 - An OAuth 2.0 *resource* to which the Office host application might need permissions.
 - An OAuth 2.0 *client* that might need permissions to Microsoft Graph.
 
+> [!NOTE]
+> The single sign-on API is currently supported in preview for Word, Excel, Outlook, and PowerPoint. For more information about where the single sign-on API is currently supported, see�[IdentityAPI requirement sets](https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets?view=office-js). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see�[Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

Comment:
For a friendlier tone, consider changing "For information about how to do this" -> "To learn how to do this"